config/v1/types_oauth.go

```diff
@@ -51,6 +51,16 @@ type TokenConfig struct {
 	// accessTokenInactivityTimeoutSeconds - DEPRECATED: setting this field has no effect.
 	// +optional
 	AccessTokenInactivityTimeoutSeconds int32 `json:"accessTokenInactivityTimeoutSeconds,omitempty"`
+
+	// accessTokenInactivityTimeout defines the default token
```

Comment:
defines the token inactivity timeout.  If you say default, people will try to find other ways to set it.

Reply:
Done.